src/PerCapita.jsx

```diff
@@ -2,35 +2,58 @@ import './PerCapita.less'
 import { GEO_NORM_NONE, GEO_NORM_PER1000 } from './constants'
 import { connect } from 'react-redux'
 import { dataNormalizationChanged } from './actions/map';
+import { hasFiltersEnabled } from './utils'
 import React from 'react'
 
 
 export class PerCapita extends React.Component {
+  _setNormalization( val ) {
+    this.props.onDataNormalization( val )
+  }
+
+  _getRawButtonClass() {
+    return this.props.dataNormalization === GEO_NORM_NONE ? '' : 'deselected'
+  }
+
+  _getPerCapButtonClass() {
+    if ( this.props.enablePer1000 ) {
+      return this.props.dataNormalization === GEO_NORM_NONE ? 'deselected' : ''
+    }
+    return 'a-btn__disabled'
+  }
   render() {
     return (
       <section className="per-capita">
-        <p>Map Shading</p>
-        <label className="u-visually-hidden" htmlFor="data-normalization">
-          Select the kind of shading to apply to the map
-        </label>
-        <select value={this.props.dataNormalization}
-                onChange={this.props.onDataNormalization}
-                id="data-normalization">
-          <option value={GEO_NORM_NONE}>Complaints</option>
-          <option value={GEO_NORM_PER1000}>Per 1000 population</option>
-        </select>
+        <div className="per-capita m-btn-group">
+          <p>Map Shading</p>
+          <label className="u-visually-hidden" htmlFor="data-normalization">
+            Select the kind of shading to apply to the map
+          </label>
```

Comment:
You should get rid of this since the drop-down disappeared.  It triggers a new WAVE warning

Reply:
thanks for validating.  Will 🔪 this